Modules/Loadable/Markups/Widgets/Resources/UI/qMRMLMarkupsFiducialProjectionPropertyWidget.ui

```diff
@@ -64,7 +64,7 @@
       </size>
      </property>
      <property name="toolTip">
-      <string>Color of the projected fiducial on 2D viewers. Only used if Use Fiducial Color is not checked, otherwise the projection uses the selected or unselected fiducial color.</string>
+      <string>Color of the projected markup on 2D viewers. Only used if Use Markup Color is not checked, otherwise the projection uses the selected or unselected markup color.</string>
```

Comment:
```suggestion
      <string>Color of the projected control points in 2D viewers. Only used if "Use Markup Color" is not checked, otherwise the projection uses the selected or unselected markup color.</string>
```